diesel/src/query_builder/insert_statement.rs

```diff
@@ -148,6 +189,140 @@ impl<T, U, Op> InsertStatement<T, U, Op, NoReturningClause> {
     }
 }
 
+#[derive(Debug, Clone, Copy)]
+/// The result of calling `insert(records).into(some_table)` when `records` is
+/// a slice or a `Vec`. When calling methods from `ExecuteDsl` or `LoadDsl`.
```

Comment:
I'm going to do a doc pass on the whole module before we ship. The `.returning` function is basically undiscoverable right now.